features/admin_center_dashboard.feature

```diff
@@ -122,9 +122,9 @@ Feature: Dashboard
     When I go to the usagov's Dashboard page
     And I follow "Manage Users"
     Then I should see the following table rows:
-      | Affiliate Manager affiliate_manager@fixtures.org            |
-      | Jane Admin admin2@fixtures.gov (Pending email verification) |
-      | John Admin admin1@fixtures.gov (Pending email verification) |
+      | Affiliate Manager affiliate_manager@fixtures.org  |
+      | Jane Admin admin2@fixtures.gov (Pending approval) |
+      | John Admin admin1@fixtures.gov (Pending approval) |
```

Comment:
This may be changed to (Approved) when I start working on SRCH-891 - Ensure admin can invite user. But for now leaving it the way it currently behaves.

Reply:
Just noting per our discussion that the approval status for approved users does not appear on the "Manage Users" page, so the scenario no longer checks for the approval status on the page.

Reply:
Just checks the users are there.